web/client/plugins/widgets/getWidgetFilterRenderers.js

```diff
@@ -35,10 +35,10 @@ const mergeQuickFiltersStream = compose(
 
 );
 
-const getFilterComponent = ({options, localType, attributeName, quickFilterStream$}) => compose(
+const getFilterComponent = ({field = {}, options, localType, attributeName, quickFilterStream$}) => compose(
     withProps({quickFilterStream$, attributeName, options}),
     mergeQuickFiltersStream
-)(getFilterRenderer(localType, {name: attributeName }));
+)(getFilterRenderer({name: field.filterRenderer?.name, type: localType, options: field.filterRenderer?.options}));
```

Comment:
you don't need anymore the attributeName  OR localType?

can they be added in OR as second condition?

Reply:
The localType is the default so it is needed. The name was never used in practice, So I removed to clean up the API